op-signer/gen-local-creds.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env sh
+
+set -euo pipefail
+
+if [ -z "$TLS_DIR" ]; then
+    SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+    TLS_DIR="$SCRIPT_DIR/tls"
+fi
+
+OPENSSL_IMAGE="alpine/openssl:3.3.3"
+
+USER_UID=$(id -u)
+USER_GID=$(id -g)
+
+CERT_ORG_NAME="OP-Signer Local Org"
+MOD_LENGTH=2048
+
+# File paths
+CA_CERT="$TLS_DIR/ca.crt"
+CA_KEY="$TLS_DIR/ca.key"
+CLIENT_TLS_KEY="tls.key"
+CLIENT_TLS_CSR="tls.csr"
+CLIENT_TLS_CERT="tls.crt"
+CLIENT_PRIVATE_KEY="ec_private.pem"
+CLIENT_OPENSSL_CNF="openssl.cnf"
+
+# Check if we should use Docker (default to true if not set)
+USE_DOCKER=${OP_SIGNER_GEN_TLS_DOCKER:-true}
+
+# Helper function to run openssl commands
+run_openssl() {
+    if [ "$USE_DOCKER" = "true" ]; then
+        docker run --rm \
+            -v "$TLS_DIR:$TLS_DIR" \
+            -u "$USER_UID:$USER_GID" \
+            "$OPENSSL_IMAGE" "$@"
+    else
+        # Check if openssl is available locally
+        if ! command -v openssl &> /dev/null; then
+            echo "Error: OpenSSL is not installed locally. Please install OpenSSL or use Docker by setting OP_SIGNER_GEN_TLS_DOCKER=true"
+            exit 1
+        fi
+        openssl "$@"
+    fi
+}
+
+generate_ca() {
+    local force="$1"
+    [ "$force" = "true" ] || [ ! -f "$CA_CERT" ] || return 0
```

Comment:
Make does this by default, which is why I was thinking it might make sense to move this to the Makefile

Reply:
it would probably be good to move to the Makefile and leverage this feature, however I'm loathe to invest in it as most OP Labs projects have transitioned to using [just](https://github.com/casey/just). could be good to migrate this one as well in a future PR